amity/Tests/test_allocations

```diff
@@ -0,0 +1,2 @@
+OFFICES...
```

Comment:
I wanted to use it to test that allocations can be printed out to a txt file but i moved it back to that files folder and forgot to remove it from the tests folder